mita/tests/unit/test_label_eval.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from mita.label_eval import (
+    pythonize_boolean,
+    validate_and_parse,
+    matching_nodes
+)
+
+boolean_exprs = {
+    'a&&b': 'a and b',
+    'a||b': 'a or b',
+    'a&&!b': 'a and not b',
+    '!a&&!b': 'not a and not b',
+    '!(a||b)': 'not (a or b)',
+}
+
+invalid_exprs = ['a.0', 'a.b', 'a.call()', '"a".len()', 'a.__dict__']
+
+
+class TestPythonize(object):
+
+    def test_valid_expr(self):
+        for expr, pyexpr in boolean_exprs.iteritems():
```

Comment:
can you please parametrize this test instead of looping? See http://pytest.org/latest/parametrize.html#parametrized-test-functions

We already use it in mita, for examples see: https://github.com/ceph/mita/blob/master/mita/tests/unit/test_util.py#L209-L233